lib/quick_travel/adapter.rb

```diff
@@ -263,7 +261,11 @@ def self.call_and_validate(http_method, path, query = {}, opts = {})
       validate!(response)
 
       if return_response_object
-        response
+        {
```

Comment:
Feels like the original purpose of this code was to return the raw response object and now we are changing it to a watered down version. Is it better to handle this in the caller?

Preexisting problem but Ideally the method should have a single consistent return value.

Reply:
Hi John, sorry for the late reply. I made changes in this method because it returns an HTTParty object which cannot be written into Rails cache either by `Rails.cache` or our Cache class in quicktravel_client. I'm not sure it is the best idea to return response body separately with headers and response code. They are what I found necessary. Do you have any thoughts regarding this? Thank you!

Reply:
Interestingly enough the bug that was causing the httpparty object to not serialise has been fixed
https://github.com/jnunemaker/httparty/pull/618/commits

Also the reason we pinned qt client to an older version has also been fixed
https://github.com/jnunemaker/httparty/pull/583